<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestCreate</class>
 <widget class="QMainWindow" name="TestCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>生成问卷</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/testIcon.png</normaloff>:/res/testIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="text_outPut">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>输出区：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outPutArea">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="chooseScope">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>251</width>
      <height>183</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>生成范围</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="A1">
       <property name="text">
        <string>生成平假名轻音部分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="A2">
       <property name="text">
        <string>生成平假名浊音部分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="A3">
       <property name="text">
        <string>生成平假名拗音部分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="B1">
       <property name="text">
        <string>生成片假名清音部分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="B2">
       <property name="text">
        <string>生成片假名浊音部分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="B3">
       <property name="text">
        <string>生成片假名拗音部分</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="fileOperation" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>471</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="text_openedFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开的文件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="openedFileOutput">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>选择文件</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/res/OpenFile.png</normaloff>:/res/OpenFile.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="KanaNumberPerLine" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="text_KanaNumberPerLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>每行假名数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="KanaNumberPerLineInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="switch_addKeys">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>文档末尾附加答案</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="chooseStyle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>生成样式</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="style1">
       <property name="text">
        <string>样式1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="style2">
       <property name="text">
        <string>样式2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="style3">
       <property name="text">
        <string>样式3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="stylePreview">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>预览样式</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToFile">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存到文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/res/saveToFile.png</normaloff>:/res/saveToFile.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/res/Create.png</normaloff>:/res/Create.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="showBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>showBg</zorder>
   <zorder>widget</zorder>
   <zorder>chooseScope</zorder>
   <zorder>line</zorder>
   <zorder>fileOperation</zorder>
   <zorder>KanaNumberPerLine</zorder>
   <zorder>switch_addKeys</zorder>
   <zorder>chooseStyle</zorder>
   <zorder>stylePreview</zorder>
   <zorder>line_2</zorder>
   <zorder>saveToFile</zorder>
   <zorder>create</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/helpIcon.png</normaloff>:/res/helpIcon.png</iconset>
   </property>
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/settingIcon.png</normaloff>:/res/settingIcon.png</iconset>
   </property>
   <property name="text">
    <string>设置背景音乐和背景</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
